src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(double radius, Color color) {
+        this.type = getClass().getSimpleName();
+        this.radius = radius;
+        this.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    public void draw() {
+        System.out.println("Figure: " + type + ", area: " + getArea()
```

Comment:
You can remove `type` field from `Figure` class and simply call `getClass().getSimpleName()` here instead of `type